Automattic/wp-calypso
# Plugin update manager: Persist queries

Closes https://github.com/Automattic/wp-calypso/issues/88513

## Proposed Changes

* Perists all the queries in plugin update manager
* Some minor changes in the spinner logic
* Fetches the query in the background while showing the old information (SWR)
* Speeds up all pages significantly
* Makes a change to [client/state/query-client.ts](https://github.com/Automattic/wp-calypso/blob/afbc872e2e56539de86806ad2040c5bc477cc8c0/client/state/query-client.ts#L28-L50) to persist query state on window unload

Was there any reason to put `{ meta: { persist: false } }` in there @bogiii ?

## Testing Instructions

1. Apply this PR
2. Check all Plugin update manager pages
3. Check for regressions across Calypso

## Pre-merge Checklist
